### PR TITLE
Moved place order to step2, added v-model for payment methods

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: view/frontend/web/js/components/BaseItems.vue [hunk 1]
```diff
@@ -35,3 +35,11 @@ export default {
...
 }
 </script>
+
+<style lang="scss" scoped>
+  .products-list {
+    margin-top: 10px;
+    padding: 10px;
+    border: 1px solid #c2c2c2;
+  }
+</style>
```
Comment (kjugi) [inline view/frontend/web/js/components/BaseItems.vue]: no components here? Just joking 😂

Comment (szafran89) [reply view/frontend/web/js/components/BaseItems.vue]: :D btw these are just temporary styles 🕺

File: view/frontend/web/js/store/index.js [hunk 2]
```diff
@@ -36,7 +36,44 @@ const store = new Vuex.Store({
...
+            "telephone": "string",
+            "postcode": "string",
+            "city": "string",
+            "firstname": "string",
+            "lastname": "string",
+            "email": "string"
+        },
+        "shipping_method_code": "string",
+        "shipping_carrier_code": "string"
+    }
```
Comment (kjugi) [inline view/frontend/web/js/store/index.js]: please delete this file b/c I see it's the same config as we have in `/templates/data/shipping-information.phtml`

Comment (kjugi) [reply view/frontend/web/js/store/index.js]: ok, I deleted it

**Pull request merged**